### PR TITLE
chore: replace warning icon with a info circle icon on connected apps list

### PR DESCRIPTION
**Pull request by helciofranco**
- Fixes tooltip title text alignment.
- Replaces that orange `warning icon` with a gray `info circle icon`.

| ⚠️  Before |  ℹ️ After |
| ---- | ---- |
| before | aftert |

### CODE REVIEW

**Review by HoTandy — APPROVED** [review]
The new grey information icon looks far less dramatic and shouldn't scare the user. Looks good.

File: examples/cra-dapp/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "cra-dapp",
+  "type": "module",
```
Comment (LuizAsFight) [inline examples/cra-dapp/package.json:3]: we need this change ?

Comment (LuizAsFight) [reply examples/cra-dapp/package.json:3]: looks orthogonal to this PR

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: examples/cra-dapp/package.json (hunk 1 above)
Comment (helciofranco) [reply examples/cra-dapp/package.json:3]: We don't really need it, but it fixes a warning when running our example dapp.

Screenshot 2024-03-29 at 19 40 09

That link recommends that we set up ESM as default in our `package.json`.
Reference: https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated

If you don't mind we can move forward with this change.

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**